### PR TITLE
Added known issue to known-issues doc

### PR DESCRIPTION
**Pull request by aleximprobable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Updated known-issues list

#### Documentation
This is documentation.

#### Primary reviewers
@oblm @m-samiec

### CODE REVIEW

File: SpatialGDK/Documentation/known-issues.md [hunk 1]
```diff
@@ -25,3 +25,4 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
...
 | Level streaming might affect performance in certain scenarios. | 2019-03-26 | [UNR-1163](https://improbableio.atlassian.net/browse/UNR-1163) | Select the **Query Based Interest Enabled** checkbox in the SpatialOS GDK for Unreal Runtime Settings to enable or disable level streaming. If the **Query Based Interest Enabled** checkbox is ticked, level streaming will be supported, but this might affect performance in certain scenarios and is currently being investigated. |
 | Changing some SpatialOS settings after launching a local deployment will invalidate the currently launched deployment (e.g. Changing the number of servers, or toggling using Query Based Interest), as these settings modify the launch configuration. | 2019-03-26 | [UNR-1190](https://improbableio.atlassian.net/browse/UNR-1190) | Manually stop and start your deployment after you make any changes that alter the launch configuration. |
+| GDK ignores the bHidden specifier for actors | 2019-04-16 | [UNR-1182](https://improbableio.atlassian.net/browse/UNR-1182) |  Use UCLASS(SpatialType = (Singleton, ServerOnly)) |
```
Comment (m-samiec) [inline SpatialGDK/Documentation/known-issues.md]: I'm not sure this workaround really solves the problem. bHidden can be toggled dynamically per actor, and marking things ServerOnly is static per class.

Comment (aleximprobable) [reply SpatialGDK/Documentation/known-issues.md]: I agree with you, should this workaround not be mentioned then?

Comment (m-samiec) [reply SpatialGDK/Documentation/known-issues.md]: Discussed offline, switch workaround to use bReplicates instead.

Comment (aleximprobable) [reply SpatialGDK/Documentation/known-issues.md]: Updated the workaround.

File: SpatialGDK/Documentation/known-issues.md [hunk 2]
```diff
@@ -25,3 +25,4 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
...
 | Level streaming might affect performance in certain scenarios. | 2019-03-26 | [UNR-1163](https://improbableio.atlassian.net/browse/UNR-1163) | Select the **Query Based Interest Enabled** checkbox in the SpatialOS GDK for Unreal Runtime Settings to enable or disable level streaming. If the **Query Based Interest Enabled** checkbox is ticked, level streaming will be supported, but this might affect performance in certain scenarios and is currently being investigated. |
 | Changing some SpatialOS settings after launching a local deployment will invalidate the currently launched deployment (e.g. Changing the number of servers, or toggling using Query Based Interest), as these settings modify the launch configuration. | 2019-03-26 | [UNR-1190](https://improbableio.atlassian.net/browse/UNR-1190) | Manually stop and start your deployment after you make any changes that alter the launch configuration. |
+| GDK ignores the bHidden specifier for actors | 2019-04-16 | [UNR-1182](https://improbableio.atlassian.net/browse/UNR-1182) |  Do not rely on bHidden, set bReplicates to the proper value instead. |
```
Comment (oblm) [inline SpatialGDK/Documentation/known-issues.md]:
```suggestion
| The GDK ignores the `bHidden` specifier for actors | 2019-04-16 | [UNR-1182](https://improbableio.atlassian.net/browse/UNR-1182) |  Do not rely on `bHidden`, set `bReplicates` to the correct value instead. |  
```

Comment (oblm) [reply SpatialGDK/Documentation/known-issues.md]: otherwise lgtm

**Pull request merged**